_dashboards/discover/index-discover.md

```diff
@@ -1,104 +1,115 @@
-1. Choose **Save**.
-1. To remove the filter, choose the close icon (x) next to the filter name.
-1. Optional: Add more filters to further explore the data.  
+### Search your data
+
+You can use the search toolbar or enter a [DQL]({{site.url}}{{site.baseurl}}/dashboards/discover/dql/) query using the **DevTools** console to search data. While the search toolbar is best for basic queries, such as searching by a field name, DQL is best for complex queries, such as searching data using a term, string, Boolean, date, range, or nested query. DQL provides suggestions for fields and operators as you type, helping you build structures queries.
+
+To search data, follow these steps:
 
-## Analyzing data in the document table
+1. Enter a simple query in the DQL search bar. For example, enter `FlightDelay:true`, which searches for delayed flights.
+1. Select the **Update** button right of the search bar.
+1. Enter a more complex query in the DQL search bar, and then select **Update**. For example, enter `FlightDelay:true AND FlightDelayMin >= 60`, which searches the data for flights delayed by 60 minutes or more.
 
-You can view the document table fields to better understand the data and gather insights for more informed decision-making: 
+### Filter your data
 
-1. Choose the arrow icon (>) to expand a table row.
-1. View the fields and details.
-1. Switch between the **Table** and **JSON** tabs to view the different formats, as shown in the following image.  
+Filters allow you to narrow down the results of a query by specifying specific criteria. You can filter by field, value, or range. The **Add filter** popup suggests the available fields and operators.
 
-![Analyzing data in the document table]({{site.url}}{{site.baseurl}}/images/discover-analyze.png)
+To filter your data:
 
-## Saving the search
+1. Under the DQL search bar, choose **Add filter**.
+1. Select the desired options from the **Field**, **Operator**, and **Value** dropdown lists. For example, select `Cancelled`, `is`, and `true`.
+1. Choose **Save**.
+1. To remove the filter, choose the cross icon ({::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/icons/cross-icon.png" class="inline-icon" alt="cross icon"/>{:/}) next to the filter name.
+1. Add more filters to further explore the data.  
+
+## Save a search
 
-Saving a search saves the query text, filters, and current data view. To save your search to use it later, generate a report, or build visualizations and dashboards:  
+To save your search, including the query text, filters, and current data view, follow these steps:  
 
-1. Choose the save icon in the toolbar. 
+1. Select **Save** in the upper-right area. 
 1. Give the search a title, and then choose **Save**. 
-1. Choose the save icon to access the saved search, as shown in the following image. 
+1. Select **Open** to access the saved search. 
+
+## Create visual representations of your data discoveries
```

Comment:
Let's try to ensure that we don't get too flowery or vague in tone. We seem to do a lot of "discovering" and "exploring" in Dashboards, when there may be a more precise description available of what the user is actually doing.

Reply:
Fair point. Replaced some instances of "exploring" and "discovering."